docs/source/user-guide/ecgtools-history-files-example.ipynb

```diff
@@ -0,0 +1,1337 @@
+    "[intake-esm](https://intake-esm.readthedocs.io/en/latest/). In this example, we use model output using the default component-set (compset) detailed in the [CESM Quickstart Guide](https://escomp.github.io/CESM/versions/cesm2.1/html/).\n",
+    "\n",
+    "## What's a \"history\" file?\n",
+    "A history file is the default output from CESM, where each file is a single time \"slice\" with every variable from the component of interest. These types of files can be difficult to work with, since often times one is interested in a time series of a single variable. Building a catalog can be helpful in accessing your data, querying for certain variables, and potentially creating timeseries files later down the road.\n",
+    "\n",
+    "Let's get started!\n",
+    "\n",
+    "## Downloading the beta version of ecgtools\n",
+    "In order to access this in its current state, you will need to clone the repository from github. On the machine with the data you are planning on creating a catalog for, run the following:\n",
+    "\n",
+    "```\n",
+    "git clone https://github.com/NCAR/ecgtools.git\n",
+    "```\n",
+    "\n",
+    "This will create a clone of the repository on your machine. After you clone the respository, run\n",
+    "\n",
+    "```\n",
+    "pip install -e ecgtools\n",
+    "```\n",
+    "\n",
+    "This will install the package into the python environment you currently activated (for more on dealing with conda environments, check out the [faq page](https://ncar.github.io/esds/faq/#conda-environments)\n",
+    "\n",
+    "You will also want to install intake-esm, which you can install using `conda-forge`\n",
+    "\n",
+    "```\n",
+    "conda install -c conda-forge intake-esm\n",
+    "```\n",
+    "\n",
+    "## Imports\n",
+    "The only parts of ecgtools we need are the `Builder` object and the `parse_cesm_history` parser from the CESM parsers! We import `glob` to take a look at the files we are parsing."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "nominated-battle",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import glob\n",
+    "\n",
+    "from ecgtools import Builder\n",
+    "from ecgtools.parsers.cesm import parse_cesm_history"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "czech-stylus",
+   "metadata": {},
+   "source": [
+    "### Understanding the Directory Structure\n",
+    "\n",
+    "The first step to setting up the `Builder` object is determining where your files are stored. As mentioned previously, we have a sample dataset of CESM2 model output, which is stored in `/glade/work/mgrover/cesm_test_data/`\n",
+    "\n",
+    "Taking a look at that directory, we see that there is a single case `b.e20.B1850.f19_g17.test`"
```

Comment:
It might be helpful to use publicly available datasets (e.g. the sample data within the repository) for this example if possible. It would make it easy for users to run and would make it easy to keep this notebook up-to-date by executing it during doc builds.

Reply:
Good point!!

Reply:
I just pushed the recent version which builds a catalog from data in the test directory